credential-info-modal-directive.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Credential Info Modal.
+ *
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+/* @ngInject */
+function factory() {
+  return {
+    restrict: 'E',
+    scope: {
+      credential: '=brCredential'
+    },
+    require: '^stackable',
+    templateUrl: requirejs.toUrl(
+      'bedrock-angular-credential/credential-info-modal.html'),
+    link: Link
+  };
+
+  function Link(scope) {
+    var model = scope.model = {};
+    model.loading = false;
```

Comment:
I can take that out but it may go back in. Modal is just showing the raw data now but I thought it might be good to add something to show data compacted with standard contexts. Since that's an async process it could use loading flags and error alerts and such. Just the raw display part is present now. Can remove the flag if it's a big deal.